extensions/amp-video/validator-amp-video.protoascii

```diff
@@ -111,10 +127,7 @@ tags: {  # <amp-video> not in amp-story.
   tag_name: "AMP-VIDEO"
   # There is an alternate spec for amp-video in amp-story.
   disallowed_ancestor: "AMP-STORY"
-  # Typically we'd require the extension j/s, but for historical reasons
-  # many pages don't have it, so it ends up late loaded and we warn, instead
-  # of making this a validation error.
-  also_requires_tag_warning: "amp-video extension script"
+  requires_extension: "amp-video"
```

Comment:
**Do not submit** until the change to `requires_extension` is merged independently.